update_ip/services/__init__.py

```diff
@@ -1,5 +1,37 @@
-from nfsn import NearlyFreeSpeechService
-from webfaction import WebFactionService
+import inspect
+import os
+import imp
+import glob
+from base import BaseDNSService
 
-services= [WebFactionService, NearlyFreeSpeechService]
```

Comment:
>I think that hardcoded services are hard to maintain
True - you have to add one element to the list every time you add a service. But I personally prefer maintaining a list with 3 elements than 30 lines of reflective code.

>if update_ip have 50 services tomorrow.
For a bit of perspective, the `update-ip` codebase is 6 years old. I added `nfsn` 5 years ago. As much as I'd love to, I don't think we're going to have 50 services tomorrow :) YAGNI